app/javascript/dashboard/routes/dashboard/settings/personal/NotificationPreferences.vue

```diff
@@ -116,6 +108,25 @@
         </div>
       </div>
     </div>
+
+    <div
+      class="flex items-center justify-between w-full gap-2 p-4 mt-3 border border-solid border-ash-200 rounded-xl dark:bg-ash-25"
```

Comment:
@muhsin-k Do we need to add the `mt-3`? I think there is a `gap-6` in the wrapper that will handle it.
```suggestion
      class="flex items-center justify-between w-full gap-2 p-4 border border-solid border-ash-200 rounded-xl dark:bg-ash-25"
```